crates/pessimistic-proof/src/multi_batch_header.rs

```diff
@@ -58,6 +58,14 @@ where
     // TODO: benchmark if BTreeMap is the best choice in terms of SP1 cycles
     pub imported_local_exit_roots: BTreeMap<NetworkId, H::Digest>,
 
+    /// The rollup exit root against which we imported the rollup bridge exits
+    #[serde_as(as = "_")]
+    pub imported_rollup_exit_root: H::Digest,
+
+    /// The mainnet exit root against which we imported the mainnet bridge exits
+    #[serde_as(as = "_")]
+    pub imported_mainnet_exit_root: H::Digest,
```

Comment:
nit: Is the `imported_*` prefix standard notation here? I would find it a bit clearer if it were only `rollup_exit_root` and `mainnet_exit_root`